interfaces/apparmor/template.go

```diff
@@ -252,6 +252,7 @@ var templateCommon = `
   /etc/{,writable/}mailname r,
   /etc/{,writable/}timezone r,
   owner @{PROC}/@{pid}/cgroup rk,
+  @{PROC}/@{pid}/cpuset r,
```

Comment:
Hmm good question - the default permissions of both files are readable by all users:

```
$ ls /proc/self/{cpuset,cgroup} -la
-r--r--r-- 1 amurray amurray 0 Feb 22 12:11 /proc/self/cgroup
-r--r--r-- 1 amurray amurray 0 Feb 22 12:11 /proc/self/cpuset
```

which then makes me think it is fine to omit the `owner` keyword  - but I suppose it would be good to be consistent (however saying that, more of the /proc/PID in the base template omit the `owner` keyword than use it):

```
[amurray:~/go … d/interfaces/apparmor] master+* ± grep '^\s*owner @{PROC}/@{pid}' template.go | wc -l
10
[amurray:~/go … d/interfaces/apparmor] master+* ± grep '^\s*@{PROC}/@{pid}' template.go | wc -l
16
```

But I am happy to add this here if you think we should.

Reply:
I have no strong opinion either way was just wondering if there was a deeper reason :)

Reply:
@alexmurray would it be worth to do a follow up to make this consistent as applicable one way or another?